hyperspy/models/model1D.py

```diff
@@ -790,8 +789,7 @@ def disable_adjust_position(self):
             if hasattr(pw, 'component'):
                 pw.component._position.twin = None
```

Comment:
This line is also no longer required, since we no longer use the hack from previously